specs/person_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../person'
+require_relative '../rental'
+
+describe Person do
+  before(:each) do
+    @p = Person.new 12
+  end
+
+  context 'Person instanitiation' do
```

Comment:
- [OBSERVATION] Did you mean: Person instantiation

Reply:
yeah